stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -417,6 +417,20 @@ extension StringProtocol where Index == String.Index {
     return self._ephemeralString as NSString
   }
 
+  func _withCore<Result>(_ apply: (_StringCore) throws -> Result) rethrows -> Result {
+    return try apply(self._emphemerialString._core)
```

Comment:
wow! it amazes me how badly I cannot spell (even when the correct spelling is just 3 lines above what I wrote...)